.github/workflows/java_api.yml

```diff
@@ -45,28 +47,55 @@ jobs:
           export ONNXRUNTIME_DIR=${PWD}
           export LD_LIBRARY_PATH=$ONNXRUNTIME_DIR/lib:$LD_LIBRARY_PATH
           popd
-      - name: Install opencv
+      - name: Install OpenCV
         run: |
           sudo apt-get install libopencv-dev
-      - name: Build java class
+      - name: Install ant
+        run: |
+          sudo apt-get install ant
+          export ANT_HOME=/usr/share/ant
+          export PATH=${ANT_HOME}/bin:${PATH}
+      - name: Build Java class
         run: |
           pushd csrc/mmdeploy/apis/java
           javac mmdeploy/*.java
           popd
-          pushd demo/java
-          javac -classpath ../../csrc/mmdeploy/apis/java/ Utils.java
-          popd
-      - name: Install mmdeploy with onnxruntime backend and java api
+      - name: Install mmdeploy with onnxruntime backend and Java API
         run: |
           mkdir -p build && pushd build
           export LD_LIBRARY_PATH=/home/runner/work/mmdeploy/mmdeploy/ncnn/install/lib/:$LD_LIBRARY_PATH
           cmake -DMMDEPLOY_BUILD_SDK=ON -DMMDEPLOY_BUILD_SDK_JAVA_API=ON -DMMDEPLOY_TARGET_BACKENDS=ort -DMMDEPLOY_CODEBASES=all -DONNXRUNTIME_DIR=~/work/mmdeploy/mmdeploy/onnxruntime-linux-x64-1.8.1  ..
           make install
+          echo $PWD
+          find /home/runner/work/mmdeploy/mmdeploy/build/lib/libmmdeploy_java.so
           popd
-      - name: Test javademo
+      - name: Prepare OpenCV Java and test Java demo
         run: |
-          export JAVA_HOME=${PWD}/jdk-18
-          export PATH=${JAVA_HOME}/bin:${PATH}
-          export LD_LIBRARY_PATH=/build/lib:${LD_LIBRARY_PATH}
-          java --version
-          python .github/scripts/test_java_demo.py
+          echo ${JAVA_HOME}
+          export OPENCV_VERSION=4.7.0
+          export JAVA_AWT_INCLUDE_PATH=${JAVA_HOME}
+          export JAVA_AWT_LIBRARY=${JAVA_HOME}
+          export JAVA_INCLUDE_PATH=${JAVA_HOME}/include
+          export JAVA_INCLUDE_PATH2=${JAVA_HOME}/include/darwin
+          export JAVA_JVM_LIBRARY=${JAVA_HOME}
+          pushd /home/runner/work
+          git clone -b ${OPENCV_VERSION} https://github.com/opencv/opencv.git
```

Comment:
It looks opencv is cloned but not built

Reply:
> It looks opencv is cloned but not built

Building OpenCV is too slow in CI, so I upload the built OpenCV libs to git lfs, and then wget to download it.

Reply:
Then why `git clone https://github.com/opencv/opencv.git`? Isn't it already built `opencv-470.jar`?